src/config/test-config.js

```diff
@@ -16,7 +16,9 @@ module.exports = {
   workerNamespace: 'worker-test-namespace',
   pipelineNamespace: 'pipeline-test-namespace',
   workerInstanceConfigUrl: 'https://raw.githubusercontent.com/biomage-ltd/iac/master/releases/production/worker.yaml',
-  pipelineInstanceConfigUrl: 'https://raw.githubusercontent.com/biomage-ltd/iac/master/releases/production/pipeline.yaml',
+  // TODO: remove this line when scaling work is merged and thus the production/pipeline.yaml is
+  // updated to new name pipelineRunner
+  pipelineInstanceConfigUrl: 'https://raw.githubusercontent.com/biomage-ltd/iac/master/releases/staging-candidates/pipeline/refs-pull-145-merge.yaml',
```

Comment:
Reminder to remov this (or create ticekt)

Reply:
same as before